package.json

```diff
@@ -133,7 +134,7 @@
     "tslint": "5.20.1",
     "tslint-react": "3.6.0",
     "typescript": "3.2.4",
-    "xhr-mock": "git://github.com/resin-io-modules/xhr-mock.git#improvements"
+    "xhr-mock": "^2.5.1"
```

Comment:
Why has this been changed in this task?

Reply:
It won't start build on PR without this change 🤷‍♂️

Reply:
Fair enough, and saying as we only use this in table ajax for the docs this is fine because we can remove it soon anyway!